mobile/library/common/jni/BUILD

```diff
@@ -56,6 +56,7 @@ cc_library(
     deps = [
         ":android_network_utility_lib",
         ":jni_utility_lib",
+        "//library/cc:engine_builder_lib",
```

Comment:
Ideally functionality common to multiple language interfaces would live in `//library/common/...` and `//library/cc` can remain a sibling to the `//library/$other_language_api` targets.

I'm happy to take on that as a clean-up item after this lands though.